packages/opentelemetry-metrics/src/Metric.ts

```diff
@@ -193,3 +198,38 @@ export class GaugeMetric extends Metric<GaugeHandle>
     this.getHandle(labelSet).set(value);
   }
 }
+
+export class MeasureMetric extends Metric<MeasureHandle>
+  implements Pick<types.MetricUtils, 'record'> {
+  protected readonly _absolute: boolean;
+
+  constructor(
+    name: string,
+    options: MetricOptions,
+    private readonly _onUpdate: Function
+  ) {
+    super(
+      name,
+      options,
+      options.valueType === types.ValueType.DOUBLE
+        ? MetricDescriptorType.GAUGE_DOUBLE
+        : MetricDescriptorType.GAUGE_INT64
+    );
+
+    this._absolute = options.absolute || true; // Absolute default is true
```

Comment:
I think `this._absolute` will be changed `true` even if I assign `false` to `options.absolute`.

```
// case 1: 
const aa = true;
console.log( aa || true); => true

// case 2: 
let aa;
console.log( aa || true); => true

// case 3: **WRONG**
const aa = false;
console.log( aa || true); => true
```

Reply:
fixed in f8dd8b6